kuma/landing/templates/landing/promote_buttons.html

```diff
@@ -284,7 +284,7 @@
         // Update the button code.
         $('#button-code').val([
             '<a href="', sel.tracked_url, '" title="', sel.alt_text, '">',
-            '<img src="{{ request.build_absolute_uri(MEDIA_URL) }}img/promote/promobutton_', sel.id, '.png" ', 
+            '<img src="{{ request.build_absolute_uri(STATIC_URL) }}img/promote/promobutton_', sel.id, '.png" ',
```

Comment:
Same as above. Unfortunately, it looks like we'll need to use `STATIC_URL` directly in some form unless we refactor this.